PocketKit/Tests/PocketKitTests/SearchViewModelTests.swift

```diff
@@ -324,13 +324,13 @@ class SearchViewModelTests: XCTestCase {
         source.stubSearchItems { _ in return savedItems }
 
         let viewModel = subject()
-        var results = viewModel.searchItems(with: "saved")
-        XCTAssertTrue(results.count == 2)
+        viewModel.submitLocalSearch(with: "saved")
+        XCTAssertEqual(viewModel.searchResults?.count, 2)
```

Comment:
is there a reason why you used `XCTAssertEqual` over the `XCTAssertTrue`?

Reply:
When I was using `XCTAssertTrue` with the optional value of `searchResults` it wanted me to unwrap it, so using `XCTAssertEqual` simplifies it. I would probably have kept it `XCTAssertTrue ` if it wasn't optional. In general, I tend to use `XCTAssertEqual` if we are checking if two expressions are equal.

Reply:
gotcha thanks